### PR TITLE
Add new presentations

### LINKED ISSUE #324: Add Percona Live Europe 2019 Presentations to resources
Extract from here: https://planetscale.com/news/planetscale-perconalive-eu-2019

Also add Jiten's talk from the Cloud Native Salt Lake City event.

### PR DESCRIPTION
**Pull request by morgo**
Fixes #324  

Waiting on slides from Jiten, and I have the Google Docs slides from Sugu but there appears to be an issue downloading and converting to PDF. I'll try again in a little bit.

Signed-off-by: Morgan Tocker <tocker@gmail.com>

### CODE REVIEW

**Comment by morgo** [comment]
I had trouble exporting as PDF from Sugu's two presentations (Google Docs timed out). What I had to do was export as LibreOffice and then convert to PDF, which broke the fonts a little.

It's not a major issue, we can swap them in the future if export works for someone else.

**Comment by deepthi** [comment]
If export as PDF fails, sometimes print as PDF will still work.

**Comment by morgo** [comment]
> If export as PDF fails, sometimes print as PDF will still work.

Thanks for the suggestion. I tried it, no luck :(

File: content/en/docs/resources/presentations.md [hunk 1]
```diff
@@ -4,6 +4,38 @@ weight: 1
...
 
+## Highload 2019
+
+Sugu Sougoumarane presents an overview of Vitess at Highload in Moscow.
+
+<iframe src = "/ViewerJS/#../files/2019-sugu-highload.pdf" width='600' height='450' allowfullscreen webkitallowfullscreen></iframe>
+
+## Utah Kubernetes Meetup 2019
+
+Jiten Vaidya shows how you can extend Vitess to create Jurisdiction-aware database clusters.
```
Comment (bnlandry) [inline content/en/docs/resources/presentations.md]: Should "Jurisdiction-aware" be capitalized? I'm guessing not.

File: content/en/docs/resources/presentations.md [hunk 2]
```diff
@@ -4,6 +4,38 @@ weight: 1
...
+
+<iframe src = "/ViewerJS/#../files/2019-jiten-utah.pdf" width='600' height='450' allowfullscreen webkitallowfullscreen></iframe>
+
+## CNCF Meetup Paris 2019
+
+Sugu Sougoumarane and Morgan Tocker present a 3hr Vitess workshop on Kubernetes.
```
Comment (bnlandry) [inline content/en/docs/resources/presentations.md]: 3hr > three hour

**Review by bnlandry — APPROVED** [review]
This page is hard to find on the site.

**Pull request merged**